Claudinee11/Property-Pro-Lite1
# Ft home page 167086263

#### What does this PR do?
-add a sign up form

#### Description of Task to be completed
- A user should be able to signup using a form and on clicking the submit button, they are redirected to sign up in page .
#### How should this be manually tested?
- clone the repo and switch to the branch ft-sign in -167087249
- enter details in field and click signup
- you should be successfully redirected to home.html
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?[#]()
[#167087249](http://pivotaltracker.com)
#### Screenshots (if appropriate)

![Capture](https://user-images.githubusercontent.com/50420731/60839824-f3d2a700-a182-11e9-9701-bac43748961c.JPG)
